pkg/controller/cluster/tenant.go

```diff
@@ -0,0 +1,73 @@
+// This file is part of MinIO Console Server
```

Comment:
Fix the license - use the wording for this project

Reply:
done